includes/wizards/class-subscriptions-wizard.php

```diff
@@ -171,35 +155,57 @@ public function register_api_endpoints() {
 			]
 		);
 	}
-
+	/**
+	 * Check whether required lugins are installed and active.
+	 *
+	 * @return bool | WP_Error True on success, WP_Error on failure.
+	 */
+	protected function check_required_plugins_installed() {
+		if ( ! function_exists( 'WC' ) || ! class_exists( 'WC_Subscriptions_Product' ) || ! class_exists( 'WC_Name_Your_Price_Helpers' ) ) {
+			return new WP_Error(
+				'newspack_missing_required_plugin',
+				esc_html__( 'The required plugins are not installed and activated. Install and/or activate them to access this feature.', 'newspack' ),
+				[
+					'status' => 400,
```

Comment:
A nitpick of pedantic proportions: Instead of 400 (Bad Request), how about [404 (Not Found)](https://httpstatuses.com/404). It's not that the client sent a bad request, but that the server couldn't find the thing the client is looking for.